docs/4.0/quickstart.md

```diff
@@ -238,14 +238,14 @@ on all servers located in Virginia:
 $ tsh --proxy=localhost ssh location=virginia ls -l /
 ```
 
-## Sharing SSH Sessions 
+## Sharing SSH Sessions
 
-Suppose you are trying to troubleshoot a problem on a node. Sometimes it makes sense to ask 
-another team member for help. Traditionally this could be done by letting them know which 
-node you're on, having them SSH in, start a terminal multiplexer like `screen` and join a 
+Suppose you are trying to troubleshoot a problem on a node. Sometimes it makes sense to ask
+another team member for help. Traditionally this could be done by letting them know which
+node you're on, having them SSH in, start a terminal multiplexer like `screen` and join a
```

Comment:
@aelkugia it looks like you have your IDE setup to remove tailing spaces. While we don't have a strong guidelines in the docs, it makes PR reviews harder to review.

Reply:
Weird the IDE setting had that unchecked. I'll keep an eye out for this in future
![whitespace](https://user-images.githubusercontent.com/8000186/65191594-5967cd00-da29-11e9-8ff5-d6099d378014.png)
 PRs.